### DISCUSSION TITLE
Axis values

### DISCUSSION THREAD

**Discussion by Renegadezz** [GSI-CS-CO/chart-fx#406] [Q&A] [answered]
Is there any way to remove the decimal from the axis values ?

**Comment by RalphSteinhagen** [comment, marked as answer]
You can customize and set your own `StringConverter<Number>` for most axes via the [tickLabelFormatterProperty()](https://github.com/GSI-CS-CO/chart-fx/blob/8fd26671ecd797718baa90fc6cb660e8489bd993/chartfx-chart/src/main/java/de/gsi/chart/axes/spi/AbstractAxisParameter.java#L1162) or if you wrote a custom axis, via providing your own implementation via the [Axis::getTickLabelFormatter()](https://github.com/GSI-CS-CO/chart-fx/blob/8fd26671ecd797718baa90fc6cb660e8489bd993/chartfx-chart/src/main/java/de/gsi/chart/axes/Axis.java#L118) interface. Hope this helps.

**Reply by Renegadezz** [reply]
Thanks!
Solution was to add this formatter to my axes:
```kotlin
    private val formatter = object : NumberStringConverter() {
        override fun toString(value: Number?): String {
            return String.format("%.0f", value)
        }
    }
```

**Reply by RalphSteinhagen** [reply]
The default is a bit more generic to handle very small/large value ranges and/or detects also, for example, when it is more suitable to switch to an exponential form. We are aware and/or unfortunately, it seems that there isn't an easy solution that fits automatically for all cases... PRs are welcome. :-)
